README.md

```diff
@@ -0,0 +1,15 @@
+# dvbtee-express
+This is a basic example of how to use node-dvbtee with udp socket.
+Thanks a lot to @mirmilad
+
+Extract EPG information and other metadata from mpegts udp stream.
```

Comment:
Please either make this general or include usage info for all of the interfaces currently supported.